src/uu/chmod/src/chmod.rs

```diff
@@ -190,16 +190,18 @@ impl Chmoder {
             let file = Path::new(filename);
             if !file.exists() {
                 if file.is_symlink() {
-                    println!(
-                        "failed to change mode of {} from 0000 (---------) to 0000 (---------)",
-                        filename.quote()
-                    );
                     if !self.quiet {
                         show!(USimpleError::new(
                             1,
                             format!("cannot operate on dangling symlink {}", filename.quote()),
                         ));
                     }
+                    if self.verbose {
+                        println!(
+                            "failed to change mode of {} from 0000 (---------) to 1500 (r-x-----T)",
+                            filename.quote()
+                        );
+                    }
```

Comment:
Not your fault, but it's weird that quiet and verbose are separate booleans isn't it? Those options should override each other, so that
```
chmod --verbose --quiet
```
is the same as
```
chmod --quiet
```

And it's also not currently possible to do, for example,
```
chmod --verbose --verbose
```

Maybe this is a good opportunity to fix that?

Reply:
Well, nevermind, that's not how GNU chmod works! `--verbose --quiet` and `--quiet` _are_ different.